packages/ui/src/composables/usePressed.ts

```diff
@@ -0,0 +1,32 @@
+import { PropType, ref, Ref, onMounted, onBeforeUnmount } from 'vue'
+
+export const usePressedProps = {
+  pressedBehaviour: { type: String as PropType<'opacity' | 'gradient'>, default: 'gradient' },
+  pressedOpacity: { type: Number, default: 0.13 },
+  pressedMaskColor: { type: String, default: '#000000' },
+}
+
+export function usePressed (el?: Ref<HTMLElement | undefined>) {
+  const isPressed = ref(false)
+
+  const onMouseDown = () => { isPressed.value = true }
+
+  const onMouseUp = () => { isPressed.value = false }
+
+  onMounted(() => {
+    if (el?.value instanceof HTMLElement) {
```

Comment:
This will not work for `nuxt-link` and `router-link`. Do we want it?

Reply:
It prevents possible errors. I'll think, how to handle nuxt/router links here.